Makefile

```diff
@@ -12,6 +12,8 @@ TAG=$(shell cat CI_TAG)
 endif
 endif
 
+QUAY_TAG_EXPIRATION=$(shell echo $(TAG) | if egrep -xq "[0-9]+(\.[0-9]+)?\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?"; then echo 24w; else echo 3w; fi)
```

Comment:
I think we should never expire releases (maybe except rc or nightlies) and for other PRs 1w will be enough.

How about simplify the regexp? We can detect if tag is from PR if it contains `x`

Reply:
Why would we want to keep an unsupported release image?
The regexp was taken from CI configuration with little adoption for `egrep`. Suggestion about `x` would indeed simplify the expression.

Reply:
I agree with @janisz that it would be best to play it safe and *not* expire releases.